book/src/installers/customizing.md

```diff
@@ -0,0 +1,109 @@
+# Customizing
+
+> since 0.12.0
+
+axo's installers are designed to require no end user input- however, in many
+cases, end users like the ability to tailor their installation experience. For
+these users, our installers allow several inputs which give the user more
+control over their installation experience via configuration. Most of these are
+implemented using environment variables, however several are also available as
+arguments to the installer scripts themselves.
+
+Several of these options were available in earlier releases of `dist`, but they
+are formally stabilized as of `0.23.0`.
+
+## Install location
+
+One of the most important jobs of an installer is to place the necessary
+artifacts in the correct location on the target machine. `dist` allows you to
+configure this for your particular needs. Depending on your setup, the following
+options are available for end users to further configure this location:
+
+- `${app name}_INSTALL_DIR`
+
+    > Replace `{app name}` with the name of the application. To transform the
+    > app name to the env var, replace any spaces or hyphens with an underscore
+    > and then put it in all caps. You can double check this transform by
+    > comparing the `install_dir_env_var` value in your `dist-manifest.json`.
+
+    This environment variable tells the installer what to use as the "base"
+    directory for the installation task. This may not necessarily be the exact
+    directory your binaries are installed to based on your install layout. For
+    example, this directory may have `./bin` appended to it.
+
+    This environment variable is also known as `CARGO_DIST_FORCE_INSTALL_DIR`. 
+
+- `$HOME`
+
+    This environment variable is sourced if you use the `CARGO_HOME` or
+    `~/sub/dir` install location options.
+
+- `${custom env var}`
```

Comment:
Likewise, link to the docs? I know what you mean here because I'm familiar with the feature, but I suspect some users won't be.